src/zenml/integrations/kubernetes/flavors/kubernetes_orchestrator_flavor.py

```diff
@@ -88,6 +90,7 @@ class KubernetesOrchestratorSettings(BaseSettings):
     failed_jobs_history_limit: Optional[NonNegativeInt] = None
     ttl_seconds_after_finished: Optional[NonNegativeInt] = None
     prevent_orchestrator_pod_caching: bool = False
+    pod_stop_grace_period: PositiveInt = 30
```

Comment:
So I'm not allowed to set it to 0? :D

Reply:
Yes, I wanted to make sure that the pod shuts down gracefully. Otherwise, it is the SIGKILL again.